go.mod

```diff
@@ -202,7 +199,7 @@ replace (
 	github.com/mholt/archiver/v3 => github.com/anchore/archiver/v3 v3.5.2
 
 	github.com/operator-framework/helm-operator-plugins => github.com/stackrox/helm-operator v0.0.12-0.20230825152000-1361e2f7db46
-	github.com/stackrox/rox => github.com/stackrox/stackrox v0.0.0-20240402171531-15fa6d254174
+	github.com/stackrox/rox => github.com/stackrox/stackrox v0.0.0-20240711164643-43f1b56f2130
```

Comment:
Is the plan to change this to the master branch before or after merging this PR?

Reply:
@RTann After merging. My plan is to merge both PRs (at the same time) and then bump the versions in both repos to use the `master` branch versions.